src/tools/kata-ctl/src/check.rs

```diff
@@ -137,6 +138,19 @@ pub enum GuestProtection {
     Se,
 }
 
+impl fmt::Display for GuestProtection {
```

Comment:
This is fine for now, but I think `GuestProtection` should be defined per platform as we do for the checks via the `get_checks()` call. I've raised #6662 so we can handle this later.